src/main/java/com/softdevsix/api/entities/Project.java

```diff
@@ -0,0 +1,81 @@
+package com.softdevsix.api.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Entity representing a project in the system.
+ * A project contains metadata such as its name and repository URL,
+ * and it can have multiple associated pull requests.
+ */
+@Getter
+@Setter
+@Entity
+public class Project {
+
+    /**
+     * The unique identifier for the project.
+     */
+    @Id
+    @GeneratedValue
```

Comment:
Please review about SQL attributes naming convention please using the framework. Besides its generation using the annotation.

Reply:
INVALID COMMENT (but fixed)
- Spring Boot with JPA not necessary for this SubTask or User Story, it was in order to help table generation for DataBase creation UserStory (CS-50 [SP-3]: Setup DB configuration for instrumenter and coverage API)
- **Solution**: Remove Spring Boot with JPA from entities